src/data-store-sql.ts

```diff
@@ -19,8 +19,17 @@ export class DataStoreSql implements DataStore {
 
     this.#db = new Kysely<DwnDatabaseType>({ dialect: this.#dialect });
 
+    // if table already exists, there is no more things todo
+    const tableName = 'dataStore';
+    const tableExists = await this.#dialect.hasTable(this.#db, tableName);
+    if (tableExists) {
+      return;
+    }
+
+    // else create the table and corresponding indexes
+
     let table = this.#db.schema
-      .createTable('dataStore')
+      .createTable(tableName)
       .ifNotExists()
```

Comment:
sanity: should this `ifNotExists` also be commented out?

Reply:
Good question, no, the `ifNotExists()` works fine with `createTable()`, just not `createIndex()`.... LOL

Reply:
💀 lol that's even more fun!!